mne/io/eeglab/eeglab.py

```diff
@@ -226,6 +233,12 @@ class RawEEGLAB(_BaseRaw):
         Names or indices of channels that should be designated
         EOG channels. If 'auto', the channel names containing
         ``EOG`` or ``EYE`` are used. Defaults to empty tuple.
+    event_id : dict
+        By default, events are read from the input file by dropping every
+        non-integer part of events containing integers, and completely
+        dropping any events without integer parts. If non-integer events
+        should be read, this should be a dict mapping from their names to
+        integers, e.g. dict(fmri_scan_onset=199, recording_start=255).
```

Comment:
> No, because for BV, most regular triggers have one of two types (style 'S 1', 'S 10' etc). But EEGLAB triggers can be arbitrary strings 
> 
But this is exactly what you want for eeglab default behavior 

So you could make both readers API consistent 
> (although I expect most to be of the same type as the BV one). There are also string-only triggers in BV files, but they should be rare (essentially recording breaks/data discontinuities and MR scan onsets are the only ones I know).
> 
> —
> Reply to this email directly or view it on GitHub.
>